rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/ArtifactEndpoint.java

```diff
@@ -163,4 +163,18 @@ Artifact getSpecific(
     Page<Build> getDependantBuilds(
             @Parameter(description = A_ID) @PathParam("id") String id,
             @BeanParam PageParameters pageParams);
+
+    @Operation(summary = "Gets the milestones that produced or consumed this artifact.",
+            responses = {
+                @ApiResponse(responseCode = SUCCESS_CODE, description = SUCCESS_DESCRIPTION,
+                    content = @Content(schema = @Schema(implementation = SwaggerPages.MilestoneInfoPage.class))),
+                @ApiResponse(responseCode = NOT_FOUND_CODE, description = NOT_FOUND_DESCRIPTION),
+                @ApiResponse(responseCode = SERVER_ERROR_CODE, description = SERVER_ERROR_DESCRIPTION,
+                    content = @Content(schema = @Schema(implementation = ErrorResponse.class)))
+    })
+    @GET
+    @Path("/{id}/milestones")
```

Comment:
what about "/{id}/info" ?

Reply:
@alexcreasy WDYT?

Reply:
I prefer {id}/milestones myself, {id}/info is too general and this is a very specific query... To me artifact/{id} is artifact "info".

It doesn't really provide more than milestones if you think about the query in userspace and not in db / implementation space.  A milestone, in reality, is a product + product version + milestone.

Person A: "Hey should we upgrade to 1.GA"? 
Person B: "Eh?"
Person A: "I mean 6.0.1.GA"
Person B: "errrrrrrm, you're talking about FUSE, right?"
Person A: "yeah, exactly. FUSE 6.0.1.GA"
Person B: "Right, sure, let's do it..."

;)

Reply:
If you would prefer something more generic the first thing that comes to my mind is "{id}/usages"

Reply:
"/{id}/milestones" is absolutely fine then, didn't know the information was gotten directly from milestone